### PR TITLE
Add Support for RedHat/CentOS 8

### PR DESCRIPTION
**Pull request by treydock**
#### Pull Request (PR) description

Ad support for RedHat / CentOS 8

#### This Pull Request (PR) fixes the following issues

This will cause conflicts with #593 so whichever is merged last will have to be rebased to fix conflicts.

### CODE REVIEW

**Comment by bastelfreak** [comment]
I vote for merging https://github.com/voxpupuli/puppet-python/pull/593 first as that has more lines in the diff so rebasing #594 is hopefully easier

**Review by bastelfreak — APPROVED** [review]

File: manifests/params.pp [hunk 1]
```diff
@@ -10,7 +10,11 @@
   }
 
   $ensure                      = 'present'
-  $version                     = 'system'
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8' {
+    $version                   = '3'
```
Comment (bastelfreak) [inline manifests/params.pp:14]: feels like we should change `$version` to `'3'` as default for all platforms. We set that in the acceptance tests and don't test anything else. (should be changed in another PR)

Comment (treydock) [reply manifests/params.pp:14]: I think the exception should be EL7 because it feels wrong to do something different than what ships with a standard install of the OS.  I think the logic could just be if EL7 , version=2 else version=3.

**Comment by treydock** [comment]
@bastelfreak I've rebased

File: manifests/params.pp (hunk 1 above)
Comment (bastelfreak) [reply manifests/params.pp]: I will try to do that in another PR
